src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
I still don't follow. Handness is a user preference and not something set by the application, hence it should not be an attribute of the component. We can emit an event whenever the controller is detected that passes the handedness for whoever needs that information from the outside.